components/script/dom/mutationobserver.rs

```diff
@@ -135,6 +136,26 @@ impl MutationObserver {
                         }
                         // Step 3.1.2
                         let paired_string = if registered.options.attribute_old_value {
+                            old_value.clone()
+                        } else {
+                            None
+                        };
+                        // Step 3.1.1
+                        let idx = interestedObservers.iter().position(|&(ref o, _)|
```

Comment:
can you rename this to `interested_observers` while at it, maybe in a different commit?

Reply:
done.